cli/src/commands/init.rs

```diff
@@ -197,13 +233,27 @@ fn prompt_lockfile_names() -> io::Result<Vec<String>> {
         };
     }
 
+    println!();
+
+    // Construct dialoguer freetext prompt.
+    let mut input = Input::new();
+    if prompt_lockfiles {
+        input.with_prompt("Other lockfiles (comma separated paths)");
+    } else {
+        input.with_prompt("Lockfiles (comma separated paths)");
```

Comment:
Why does there need to be more context? This message provides no value imo.

Reply:
If they have previously used the CLI and it did autodetect some lockfiles, and then a subsequent run in another project takes them directly to a manual entry (and no choice to select) it might be slightly confusing. The text helps clarify why they've found themselves in this state.

Reply:
Is that not more of an implementation detail? If we find no groups, we also do not notify the user that he has no groups but we just skip the step entirely.

Would you suggest also adding a message stating that group selection was skipped since no group was found?

If we want to do this, we should be consistent. And I'm just concerned that this level of verbosity consistently will create too much text.

Reply:
Ooh...adding a message about how no groups were found does sound good.

The _interactive_ use of `phylum init` feels like an acceptable interface for extra text and hand-holding. Power users can always specify everything with arguments directly and avoid the interactive nature entirely.

Reply:
> Power users can always specify everything with arguments directly and avoid the interactive nature entirely.

This forces power users to type out the entire command and lose the advantage of the defaults though. Meaning the *power* user, so the person using it the most, needs to put in more effort to get the same result. That seems backwards to me.

Reply:
> This forces power users to type out the entire command and lose the advantage of the defaults though.

Good point. We might be better served to go through this review as a group in real time, to include at least @furi0us333 and maybe @louislang since he/they are interacting directly with users at this time.

Reply:
`No known lockfiles were found in the current directory. Please manually enter at least one.`

Reply:
Group consensus:
`No known lockfiles found in current directory. Please enter at least one.`
```suggestion
        input.with_prompt("Lockfile(s) (comma separated paths)");
```